README.md

```diff
@@ -2208,6 +2253,8 @@ Frontiers in Neuroscience, 2022.
 *[Astrometric Calibration and Source Characterisation of the Latest Generation Neuromorphic Event-based Cameras for Space Imaging](https://arxiv.org/abs/2211.09939)*, 
 arXiv preprint arXiv:2211.09939, 2022. 
 [Dataset](https://github.com/NicRalph213/ICNS_NORALPH_Event_Based-Space_Imaging-Speed_Dataset).
+- [Mahlknecht et al., RAL 2022](#mahlknecth22ral),  
```

Comment:
link is missing

Reply:
Not sure which link is missing?

Reply:
When you display the rich diff, preview, does it appear like a link? or because the different capitalization of the anchor, "Mahlknecth22ral" vs "mahlknecth22ral" no link is created?

Reply:
![image](https://user-images.githubusercontent.com/8024432/229792478-b5d76c60-af81-47f8-ac54-4cf2e1b63e40.png)
not blue -> no link